### PR TITLE
Update Server Docs

### PR DESCRIPTION
**Pull request by digisomni**
This PR does not correct all discrepancies in the server documentation, it's just one piece of many to help bring it up to spec.

Based on PR #136

### CODE REVIEW

**Comment by JulianGro** [comment]
Can you rebase this on master?

**Comment by digisomni** [comment]
Got'em.

File: docs/source/host/server-setup/windows-server.rst [hunk 1]
```diff
@@ -1,14 +1,10 @@
-.. warning::
-    This document is outdated.
-    FIXME: mentions purchasing domain names, only mentions DigitalOcean
-
-#################################
-Host a Domain from a Local Server
-#################################
+#####################################
+Host a Domain from Locally on Windows
```
Comment (JulianGro) [inline docs/source/host/server-setup/windows-server.rst:2]:
```suggestion
Host a Domain locally on Windows
```

File: docs/source/host/server-setup/windows-server.rst [hunk 2]
```diff
@@ -34,17 +30,17 @@ To set up a domain and host it on your local server:
...
     
-    .. note:: Your temporary place name is only active while your domain is being staged. Permanant place names can be purchased on a subscription basis, and are renewed yearly. :doc:`Learn how to purchase a custom place name for your domain <../configure-settings/place-setting>`. 
+    .. note:: Learn :doc:`how to configure a Place name <../configure-settings/place-setting>`.
     
-    * **Step 3: Set up basic permissions**: By default, anyone logged on to your computer will have access to the domain settings for your domain. If you would like to extend administrator privileges to remote users (or allow yourself access from a remote location), then add the Vircadia usernames here. In addition, choose who will be able to connect or rez items. 
+    * **Step 3: Set up basic permissions**: By default, anyone logged on to your computer will have access to the domain settings for your domain. If you would like to extend administrator privileges to remote users (or allow yourself access from a remote location), then add the Vircadia Metaverse usernames here. In addition, choose who will be able to connect or rez items.
```
Comment (JulianGro) [inline docs/source/host/server-setup/windows-server.rst:43]:
```suggestion
    * **Step 3: Set up basic permissions**: By default, anyone logged on to your computer will have full access to your domain. If you would like to extend administrator privileges to remote users (or allow yourself access from a remote location), then add the Vircadia Metaverse usernames here. In addition, choose who will be able to connect or rez items. 
```

File: docs/source/host/server-setup/windows-server.rst [hunk 3]
```diff
@@ -59,9 +55,9 @@ To activate your domain:
...
 
-.. note:: If you did not purchase a permanent place name, the only way to access your domain is by your server's IP address. However, note that when your IP address changes, so too will the address of your domain and you will need to manually restart your domain each time you change networks. Therefore, if you are hosting your domain on a laptop or on a computer with a frequently changing IP address, we strongly recommend that you purchase a permanent place name for your domain. :doc:`Learn how to purchase a custom place name for your domain <../configure-settings/place-setting>`. 
+.. note:: If you did not assign a Place name, the only way to access your domain is by your server's IP address. However, note that when your IP address changes, so too will the address of your domain and you will need to manually restart your domain each time you change networks. Therefore, if you are hosting your domain on a laptop or on a computer with a frequently changing IP address, we strongly recommend that you purchase a permanent Place name for your domain. Learn :doc:`how to configure a Place name <../configure-settings/place-setting>`.
```
Comment (JulianGro) [inline docs/source/host/server-setup/windows-server.rst:60]:
```suggestion
.. note:: If you did not assign a Place name, the only way to access your domain is by your server's IP address. However, note that when your IP address changes, so too will the address of your domain. Therefore, if you are hosting your domain on a laptop or on a computer with a frequently changing IP address, we strongly recommend that you configure a Place name for your domain. Learn :doc:`how to configure a Place name <../configure-settings/place-setting>`. 
```

File: docs/source/host/start-hosting.rst [hunk 4]
```diff
@@ -17,19 +13,19 @@ Overview of Hosting
...
-* A *domain* is a registered location in the metaverse. Each domain is assigned a unique ID to identify it. You can sign up for a new domain in one of two ways: either by setting up a Vircadia server or by logging into your account via the Vircadia website.
-* A *server* hosts a domain, and processes incoming network requests for the domain. The server for your domain is either a physical computer or a cloud-hosted server.
-* A *droplet* or *cloud domain* is a cloud server hosted on DigitalOcean.
-* A *content set* is the content that you've designed to make your domain unique and pretty. 
+* The *Domain Server* is a virtual world that gives you a space to build content in.
+* A *domain* is a registered server in the metaverse. Each domain is assigned a unique ID to identify it. You can connect your domain to the metaverse by logging into your account via the Vircadia website.
```
Comment (JulianGro) [inline docs/source/host/start-hosting.rst:17]:
```suggestion
* A *domain* is a server in the metaverse. Each domain is assigned a unique ID to identify it. You can connect your domain to a Metaverse Server.
```

**Review by JulianGro — CHANGES_REQUESTED** [review]

File: docs/source/host/start-hosting.rst (hunk 4 above)
Comment (digisomni) [reply docs/source/host/start-hosting.rst:17]: I think that the specific reference to the Vircadia website will make the most sense until we have federation working and UX can easily make obvious how it all connects.

Comment (JulianGro) [reply docs/source/host/start-hosting.rst:17]:
```suggestion
* A *domain* is a server in the metaverse. Each domain is assigned a unique ID to identify it. You can connect your domain to the Vircadia Metaverse Server by logging into your account on the Vircadia website.
```
Yeah I guess so. How about this then?

**Review by JulianGro — APPROVED** [review]

**Pull request merged**